packages/wrapper/src/editorAppExtended.ts

```diff
@@ -106,6 +108,8 @@ export class EditorAppExtended extends EditorAppBase {
     disposeApp(): void {
         this.disposeEditors();
         this.extensionRegisterResults.forEach((k) => k?.dispose());
+        this.subscriptions.forEach((k) => k.dispose());
+        this.subscriptions.length = 0;
```

Comment:
nit: for informations (@kaisalmen ) `vcode/monaco` export the `DisposableCollection` class from VSCode, do not hesitate to use it :)

Reply:
Thanks, I've updated it to use `DisposableCollection`. I wasn't able to find it in the `vscode` or `monaco-editor` packages, but I did find it in the `vscode-ws-jsonrpc` package so I used it from there.

Reply:
@CGNonofr you meant `import { DisposableStore } from 'vscode/monaco';` like used here https://github.com/CodinGame/monaco-vscode-api/blob/main/src/service-override/tools/views.ts#L258-L367, right?